crates/egui/src/widgets/plot/mod.rs

```diff
@@ -331,6 +332,7 @@ impl Plot {
             show_background: true,
             show_axes: [true; 2],
             grid_spacers: [log_grid_spacer(10), log_grid_spacer(10)],
+            sharp_grid_lines: false,
```

Comment:
We can discuss whether to make this the default or not. I personally don't mind the slight aliasing.

Reply:
I think I prefer it on (as is currently the case)

Reply:
Now enabled by default in https://github.com/emilk/egui/pull/2412/commits/848310deeb33cc9b073722cadde41e828bdea450.